onboarding/geo/README.md

```diff
@@ -0,0 +1,334 @@
+- [What is GIS?](https://www.youtube.com/watch?v=Mw2KH1WQ8oA)
+- [National Geographic Article](https://education.nationalgeographic.org/resource/geographic-information-system-gis/)
+- [Free Introductory Book](https://volaya.github.io/gis-book/en/index.html)
+- [UC Davis GIS Specialisation](https://www.coursera.org/specializations/gis)
+- [Video on Geo-Spatial Data Concepts](https://www.youtube.com/watch?v=4LVsYa7GVDE)
+
+## An Introduction to Satellite Imagery
+
+### Introduction
+
+Satellite imagery is a powerful and invaluable resource that provides us with a unique perspective on our planet. From monitoring climate patterns to studying land use changes and tracking natural disasters, satellite imagery plays a crucial role in a wide range of fields, including environmental science, agriculture, urban planning, and disaster management. In this section, we'll explore the world of satellite imagery, understand the data it offers, and introduce you to some of the most commonly used NASA and ESA data sets.
+
+### Data Associated with Satellite Imagery
+
+Satellite imagery consists of high-resolution images captured by satellites orbiting Earth. These images are generated using various sensors and instruments that capture data across different parts of the electromagnetic spectrum, including visible light, infrared, and microwave wavelengths. The data associated with satellite imagery provides a wealth of information:
+
+- **Spectral Data:** Different sensors capture data in different spectral bands, allowing us to analyze phenomena such as vegetation health, land cover, and atmospheric composition.
+
+- **Temporal Data:** Satellites capture images at regular intervals, providing a historical record of changes over time. This is essential for tracking trends, monitoring seasonal variations, and assessing long-term impacts.
+
+- **Geospatial Data:** Satellite imagery includes geospatial information, enabling precise georeferencing and mapping. This allows us to correlate imagery with specific geographic locations on Earth.
+
+- **Resolution:** Satellite imagery comes in various resolutions, from low-resolution global views to high-resolution images that can capture fine details on the Earth's surface.
+
+## An Introduction to Satellite Bands
+
+To make the most of satellite imagery, it's essential to understand the concept of satellite bands. Satellite bands refer to the specific regions of the electromagnetic spectrum that satellites capture and record. Each band provides unique information about the Earth's surface and atmosphere, making them essential for various geospatial analyses.
+
+### Understanding Satellite Bands
+
+Satellite sensors are equipped with detectors that measure electromagnetic radiation in different parts of the spectrum. Here's an overview of the key satellite bands and their characteristics:
+
+### 1. **Visible Bands:**
```

Comment:
Really nice work on explaining bands. The norm is to normalize everything to nanometers instead of micrometers. It saves having the 0.45 micrometer,  in nanometers it becomes 450 nm.